README.md

```diff
@@ -19,6 +19,16 @@ $ cpm [i|r|l|u|U|s|S|c|h] [pkg]...
 -> h|help    show this message
 ```
 
+## Installation:
```

Comment:
Also, remove the colon from this line, like with Usage. In fact, I'd appreciate if you removed all the colons on the header lines, not sure why I put them there in the first place.